### ISSUE TITLE
Not installing if project react version >= 17.0.1

### ISSUE THREAD

**Issue by therezacuet** [DylanVann/react-native-fast-image#793]
I tried to install this package on my project but I got a conflict error for react version. Do you have any plan to upgrade peer dependency version?

**Comment by kishanbharda** [comment]
I have also an error while installing this library in my latest react-native project.

My project configuration : 
```
"react": "17.0.1",
"react-native": "0.64.2",
```

This is the error that I am gettings : 
![Screenshot 2021-06-10 at 4 58 36 PM](https://user-images.githubusercontent.com/39869106/121517905-81713e00-ca0d-11eb-9b0d-13d874096afc.png)

**Comment by hellotoby** [comment]
You can solve this by running `npm install --legacy-peer-deps`

**Comment by kishanbharda** [comment]
Thank @hellotoby 

Installing this package with `--legacy-peer-deps` solved my problem. I have installed with below command and it installed :
```
npm install react-native-fast-image --legacy-peer-deps --save
```

Does the package install this way, can create any other issues?

**Comment by Desintegrator** [comment]
+1

**Comment by Desintegrator** [comment]
@kishanbharda is it working this way?

**Comment by therezacuet** [comment]
@Desintegrator yes this package working this way.

**Comment by dcx86** [comment]
> _Fix the upstream dependency conflict, or retry
> this command with --force, or --legacy-peer-deps
> to accept an incorrect (and potentially broken) dependency resolution._

Running `npm install react-native-fast-image --legacy-peer-deps --save` will not  solve the potentially broken dependency resolution it will just ignore them.
 
Do you have any plan to upgrade peer dependency version?

**Comment by DEVfancybear** [comment]
+1

**Comment by sanduluca** [comment]
+1

**Comment by naitikgohain** [comment]
Issue fixed in #790 merge, and the fix is in version 8.3.5 as per https://github.com/DylanVann/react-native-fast-image/pull/790#issuecomment-875101311
